pyproject.toml

```diff
@@ -122,6 +122,7 @@ dependencies = [
     # - `ibm-cos-sdk-core` requires `requests<2.32.3,>=2.32.0`
     # - `reportportal-client` requires `requests>=2.32.3`
     "requests==2.32.2",
+    "transformers==4.50.3",
```

Comment:
Can you add a comment why we need to pin it? To better understand when we can unpin again. :)